traffic_portal/app/src/common/modules/table/deliveryServiceRequests/TableDeliveryServiceRequestsController.js

```diff
@@ -136,12 +142,15 @@ var TableDeliveryServicesRequestsController = function (tableName, dsRequests, $
 		onRowClicked: function(params) {
 			const selection = window.getSelection().toString();
 			if(selection === "" || selection === $scope.mouseDownSelectionText) {
-				let path = '/delivery-service-requests/' + params.data.id + '?type=';
-				typeService.getType(params.data.deliveryService.typeId)
+				let path = '/delivery-service-requests/' + params.data.id + '?type=',
+					typeId = (params.data.requested) ? params.data.requested.typeId : params.data.original.typeId;
```

Comment:
`typeId` is never reassigned, could be `const` - also one declaration at a time is easier to read